docs/pages/desktop-access/getting-started.mdx

```diff
@@ -176,6 +192,24 @@ Computer Configuration > Policies > Windows Settings > Security Settings > Publi
   ![Import Teleport CA](../../img/desktop-access/ca.png)
 </Figure>
 
+### Publish the Teleport CA to the Active Directory domain
+
+<Admonition type="note" title="Differences when using AWS Managed Active Directory">
+  When using AWS Managed Active Directory, you should run this command using an account which is part of the
+  `AWS Delegated Domain Administrators` group, such as the AWS-provided `admin` account.
+</Admonition>
+
+On a machine which is joined to your domain and logged in as an account in the `Domain Administrators` group,
+run the two commands below at a PowerShell prompt to publish the Teleport CA to your Active Directory
+domain (using the path to the exported Teleport `user-ca.cer` file that you copied above):
+
+```powershell
+certutil –addstore –f root C:\Path\To\user-ca.cer
+certutil –dspublish –f C:\Path\To\user-ca.cer RootCA
```

Comment:
I do have a more detailed section of terminal output here with real outputs which might help guide people better:

```powershell
PS C:\temp> certutil -addstore -f root C:\temp\user-ca.cer
root "Trusted Root Certification Authorities"
Signature matches Public Key
Certificate "teleport-cluster" added to store.
CertUtil: -addstore command completed successfully.

PS C:\temp> certutil -dspublish -f C:\temp\user-ca.cer RootCA
ldap:///CN=teleport-cluster,CN=Certification Authorities,CN=Public Key Services,CN=Services,CN=Configuration,DC=teleportdemo,DC=com?cACertificate

Certificate added to DS store.

ldap:///CN=teleport-cluster,CN=AIA,CN=Public Key Services,CN=Services,CN=Configuration,DC=teleportdemo,DC=com?cACertificate

Certificate added to DS store.

CertUtil: -dsPublish command completed successfully.
```

I feel like it makes the commands much harder to read, though, so chose to just include those two in the docs.

@klizhentas @ptgott Any thoughts?

Reply:
I would stick with what you have, for the reason you stated. That's also how we're doing it elsewhere in this doc, some of the commands have crazy long outputs.